integration-tests/ci-visibility/test-api-manual/run-fake-test-framework.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+require('./setup-fake-test-framework')
```

Comment:
Is this `require` necessary? You are doing a require with `--require` here: https://github.com/DataDog/dd-trace-js/pull/3202/files#diff-2d02c6c61a37789299433916e8654ada993c0f6b43f2a881c0ef8019f4227269R71

Reply:
yeah that's a good point. Likely not

Reply:
removed it in https://github.com/DataDog/dd-trace-js/pull/3202/commits/d7768fa1687dd12d86033ddc978e219c154fafca